Dashboard/components/simulation/Blockchain.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box } from "@mantine/core";
+import { useEffect, useState } from "react";
+import { Subscription } from "../../web3/newHeads";
+import Block from "./elements/Block";
+
+interface PropTypes {
+  hashes?: string[];
+  nodes?: string[];
+}
+export const Blockchain = (nodes: PropTypes) => {
+  const [blocks, setBlocks] = useState([]);
+  const [index, setIndex] = useState(0);
+
+  useEffect(() => {
+    Subscription.on("data", (data) => {
+      const block = (
+        <div style={{ display: "flex", flexDirection: "column" }}>
+          {index > 0 && (
+            <svg width="100" height="55">
+              <line
+                x1="55"
+                y1="0"
+                x2="55"
+                y2="55"
+                stroke="black"
+                stroke-width="2"
+              ></line>
+            </svg>
+          )}
+          <Block key={data.hash} data={data}></Block>
+        </div>
+      );
+
+      setIndex(index + 1);
+      setBlocks([...blocks, block]);
```

Comment:
I think this may be why we are getting list errors in the console. Make sure that each block has a unique key and two blocks are not being generated. UseEffect triggers on all changes to the state, so changing the state within useEffect can cause issues. 

In the console you will see that the newHeader object is logged twice.